src/api.ts

```diff
@@ -109,6 +112,10 @@ export const apiFactory = (serviceId: number) => {
 			return res.sendStatus(400);
 		}
 
+		if (workerMap[req.body.common_name] !== req.params.worker) {
+			return res.sendStatus(400);
```

Comment:
Ignore this comment if it is not relevant -- I've jumped in this PR without much context / knowledge of how the VPN service works. But it looks like workerMap is a simple object / dictionary local to the server handling the request. If the requests for `api.post('/api/v2/:worker/clients/')` and `api.delete('/api/v2/:worker/clients/')` were served by different machines (load balancing), then the delete request would be answered with 400.

Reply:
@pdcastro almost, the `worker id` refers to a process on the machine, not a different physical machine.

Reply:
But if there were multiple physical machines (maybe there aren't), the issue could still happen... Anyway, it sounds like you're well aware of it, so approving the PR. :-)